src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+  const key = Object.keys(data);
+  const values = Object.values(data);
+
+  for (let i = 0; i < key.length; i++) {
+    const lists = document.createElement('li');
```

Comment:
```suggestion
    const listItem = document.createElement('li');
```
Это один элемент, и не список, а элемент списка